src/configure.ts

```diff
@@ -16,13 +18,49 @@ export function jestPreviewConfigure(
 
   options.externalCss.forEach((cssFile) => {
     const basename = path.basename(cssFile);
+
     // Avoid name collision
     // Add `global` to let `jest-preview` server that we want to cache those files
     const destinationBasename = `cache-${basename}`;
-    const destinationFile = path.join(
-      './node_modules/.cache/jest-preview-dom',
-      destinationBasename,
-    );
+    const destinationFile = path.join(CACHE_FOLDER, destinationBasename);
+
+    // If sass file is passed, we need to transform it to css
+    if (cssFile.endsWith('.scss') || cssFile.endsWith('.sass')) {
+      const sass = require('sass');
+
```

Comment:
This is not safe since user might not have `saas` installed locally. Might need to try catch

Reply:
Yeah, also, `sass-loader` supports multiple sass implementation (node-sass, dart-sass,...), maybe we can refer to that